deeplearning4j-modelimport/pom.xml

```diff
@@ -125,5 +125,19 @@
             <artifactId>logback-classic</artifactId> <!-- Version set by deeplearning4j-parent dependency management -->
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.deeplearning4j</groupId>
+            <artifactId>deeplearning4j-core</artifactId>
+            <version>0.7.1-SNAPSHOT</version>
```

Comment:
Version is out of date. ALWAYS use ${project.version} Please do a git pull and update your versions if you haven't already.